package.json

```diff
@@ -86,7 +93,7 @@
 			"strict": "error",
 			"ava/no-ignored-test-files": "off",
 			"eslint-plugin/prefer-message-ids": "off",
-			"eslint-plugin/require-meta-docs-description": "off",
+			"eslint-plugin/require-meta-docs-description": ["error", { "pattern": "^(Enforce|Ensure|Require|Disallow|Prevent|Prefer)" }],
```

Comment:
We now enforce that rule descriptions are stored inside the standard `meta.docs.description` field in rules. This allows tooling like eslint-doc-generator to utilize these descriptions. I made the same change in eslint-plugin-unicorn last year in https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1155.